index.html

```diff
@@ -257,6 +277,34 @@
 					document.getElementById(counterID).innerText = pointsInDistrict.length;
 				}
 			}
+
+			function standardizeCase(txt) {
+				console.log("INPUT", txt);
+				if (txt === undefined || txt === "") {
+					return "";
+				} else {
+					txt = txt.replace(/([^\W_]+[^\s-]*) */g, function(x) {
+						return x.charAt(0).toUpperCase() + x.substr(1).toLowerCase();
+					});
+					txt = " " + txt + " ";
+	// NB: always include the leading & trailing spaces in this list to avoid accidentally selecting substrings.  The .trim() call at the end will clean them back up.
+					overrides = [
```

Comment:
This array of arrays is the list of everything that the function will either expand acronyms for or keep as all-caps.  I recommend showing this list to the client, and asking them if there are any they'd like to add or have treated differently.

Reply:
Great idea. I will send the list on to Joel & Max today. Nice work on this by the way!